atlas/rules/assemble.snakefile

```diff
@@ -108,7 +108,7 @@ rule error_correction:
         temp(expand("{{sample}}/assembly/reads/{{previous_steps}}.errorcorr_{fraction}.fastq.gz",
             fraction=MULTIFILE_FRACTIONS))
     benchmark:
-        "logs/benchmarks/assembly/pre_process/error_correction_{previous_steps}/{sample}.txt"
+        "logs/benchmarks/assembly/pre_process/{sample}_error_correction_{previous_steps}.txt"
```

Comment:
The rationale behind my naming was that in the end you can concat all files in the same subfolder and take the average or max, and get the information you want. 

But for now I haven't used the benchmark files.